velox/functions/sparksql/String.cpp

```diff
@@ -103,6 +103,229 @@ class Length : public exec::VectorFunction {
   }
 };
 
+void doApply(
```

Comment:
I feel we are lacking some description for the process of concating, maybe better to add some.